app/javascript/shared/components/useKeyboardShortcuts.js

```diff
@@ -12,22 +12,51 @@ function isFormField(element) {
   if (element instanceof HTMLElement === false) return false;
 
   const name = element.nodeName.toLowerCase();
-  const type = (element.getAttribute('type') || '').toLowerCase();
+  const type = (element.getAttribute("type") || "").toLowerCase();
   return (
-    name === 'select' ||
-    name === 'textarea' ||
-    (name === 'input' &&
-      type !== 'submit' &&
-      type !== 'reset' &&
-      type !== 'checkbox' &&
-      type !== 'radio') ||
+    name === "select" ||
+    name === "textarea" ||
+    (name === "input" &&
+      type !== "submit" &&
+      type !== "reset" &&
+      type !== "checkbox" &&
+      type !== "radio") ||
     element.isContentEditable
   );
 }
 
+/**
+ * Function to handle 
+ * 
+ * @param {KeyboardEvent} e Keyboard event
+ * @param {String} keys special keys formatted in a string
+ * @param {Array} chain array of past keys
+ * @param {Object} shortcuts object containing callback functions
+ */
+const callShortcut = (e, keys, chain, shortcuts) => {
+  let shortcut =
```

Comment:
```suggestion
  const shortcut =
```
refactor (blocking) - we don't reassign this variable so lets make it a `const`

Reply:
@aitchiss, I'm going to add the [prefer-const](https://eslint.org/docs/rules/prefer-const)eslint rule to the project so this gets picked up during linting.

Reply:
See #13098